src/test/js/App.test.js

```diff
@@ -0,0 +1,7 @@
+
+
+describe('Example tests', () => {
+  it('Check 1+2 equals the correct number', () => {
+    expect(1 + 2).toBe(3);
```

Comment:
Please add more tests  then just this one to the PR

Reply:
This is an initial example as proof of how to start js tests with react and jest.
It was there to get people going with learning jest.
Other tests are added later.